mysql-test/suite/innodb_gis/t/stats_wo_innodb_table_stats.test

```diff
@@ -0,0 +1,23 @@
+#
+#  MDEV-24558 Assertion `index->is_btree()` failed
+#
+
+-- source include/have_innodb.inc
+-- source include/have_debug.inc
+
+-- disable_warnings
```

Comment:
Usually, the `--source` statements are at the start of the test. A test specific header is usually written with three lines that start with `--echo #`.

There is no real need for the `have_debug.inc`. The test could run also on `CMAKE_BUILD_TYPE=RelWithDebInfo`, as far as I can tell. Could this test be added to an existing test file?